integration-tests/runtest.sh

```diff
@@ -3,7 +3,6 @@
 if [ ! -f /var/run/docker.pid ]
 then
     echo "!!! Docker service is probably not running !!!"
-    exit 1
```

Comment:
if the docker is not running, then when running locally the tests will fail later a bit later, not immediately. with this change we can run the tests against openshift and docker doesn't have to be running locally.